drone-ci/drone/ansible/play.yml

```diff
@@ -118,3 +124,15 @@
         env_file: "{{ drone_local_dir_path }}/dronerc"
         published_ports:
           - 80:8000
+    - name: run drone agent
```

Comment:
Not sure all about drone. But, if it is something that should be run after successful installation/setup of drone then its safe to keep it in handler.